r/vignettes/fs.Rmd

```diff
@@ -13,7 +13,10 @@ parts of the project to be able to read and write data with different storage
 backends. In the `arrow` R package, support has been enabled for AWS S3.
 This vignette provides an overview of working with S3 data using Arrow.
 
-> In Windows and macOS binary packages, S3 support is included. On Linux when installing from source, S3 support is not enabled by default, and it has additional system requirements. See `vignette("install", package = "arrow")` for details.
+> In Windows and macOS binary packages, S3 support is included. On Linux when 
+installing from source, S3 support is not enabled by default, and it has 
+additional system requirements. See `vignette("install", package = "arrow")` 
+for details.
```

Comment:
No changes made here, I was just breaking it up across multiple lines like the rest of the paragraphs

Reply:
🙏 for both the line breaks + proactively clarifying